Application/src/visuals/Message.java

```diff
@@ -7,19 +7,23 @@
 
 /**
  * Class for creating the messages to pass between "classes".
+ *
  * @author Sebastian Fransson
- * @version 1.0
+ * -collaborator Rashad Kamsheh
+ * @version 1.1
  */
-public class Message implements Renderable{
+public class Message implements Renderable {
     private String name;
-    private Coordinates coordinates , node1, node2; // The coordinates of the nodes that the message is supposed to pass between.
+    private Coordinates coordinates, node1, node2; // The coordinates of the nodes that the message is supposed to pass between.
     private int fromNode, toNode;
     private int offset; // offset for the message "ordering".
     private int class_size; // identifier for the current class size so that messages can scale.
     private int animationBounds = 0; //Starts and stops the animation at the correct bounds.
     private boolean keepAnimating = true; // State of the animation. Beginning or ending.
     private boolean switchImage; // Keeps track of which image to show.
     private double messageScale = 1.5;
+    private int stage = 1; // Used for the hardcoded trajectory of self referencing message.
```

Comment:
Why is it called stage? Is it the same kind of stage we are currently using in javafx?

Reply:
no, it should be renamed